modules/gltf/doc_classes/GLTFTexture.xml

```diff
@@ -10,7 +10,7 @@
 		<member name="sampler" type="int" setter="set_sampler" getter="get_sampler" default="-1">
 			ID of the texture sampler to use when sampling the image. If -1, then the default texture sampler is used (linear filtering, and repeat wrapping in both axes).
 		</member>
-		<member name="src_image" type="int" setter="set_src_image" getter="get_src_image" default="0">
+		<member name="src_image" type="int" setter="set_src_image" getter="get_src_image" default="-1">
 		</member>
```

Comment:
I didn't realize we missed this description for the property src_image.

The idea is this image is useful for accessing the original image before other extensions have tinkered with it.

I doubled checked we needed this in a VRM portable avatar extension for generating the Mtoon shader images for the materiak.

Reply:
Note that `GLTFTexture::src_image` just has the index of the image saved in `GLTFState`. You are probably thinking of `GLTFState::source_images` when you wrote this comment but note that this is not accessible from extension, only `GLTFState::images` is accessible (and with `GLTFState::images` you can do everything that you can with `GLTFState::source_images`, thus why in the initial version of this PR I removed the latter but you told me to keep it).